app/src/main/java/com/conupods/OutdoorMaps/MapInitializer.java

```diff
@@ -76,4 +66,23 @@ public void initializeBuildingMarkers() {
             return false;
         });
     }
+
+    public static void setSearchViewOnClickListener(View v, View.OnClickListener listener) {
+        if (v instanceof ViewGroup) {
```

Comment:
why dont u pass a ViewGroup instead of using instanceof

Reply:
the search view is comprised internally of many different types of objects. Im checking to see if any are instances of ViewGroups. So i use instanceOf